src/convenience/keyboard.ts

```diff
@@ -112,6 +139,49 @@ export class Keyboard {
     webApp(text: string, url: string) {
         return this.add({ text, web_app: { url } });
     }
+    /**
+     * Make current keyboard selective.
+     * See https://grammy.dev/plugins/keyboard.html#sending-a-custom-keyboard
+     * for more detail.
+     *
+     * @param isEnabled Determine if the keyboard should be selective. Default to true.
+     */
+    selected(isEnabled = true) {
+        this.selective = isEnabled;
+        return this;
+    }
+    /**
+     * Make current keyboard one time.
+     * See https://grammy.dev/plugins/keyboard.html#sending-a-custom-keyboard
+     * for more detail.
+     *
+     * @param isEnabled Determine if the keyboard should be one time. Default to true.
+     */
+    oneTime(isEnabled = true) {
+        this.one_time_keyboard = isEnabled;
+        return this;
+    }
+    /**
+     * Make current keyboard resizeable.
+     * See https://grammy.dev/plugins/keyboard.html#resize-custom-keyboard
+     * for more detail.
+     *
+     * @param isEnabled Determine if the keyboard should be resizeable. Default to true.
```

Comment:
@roj1512 Do you have any suggestion about this?

Reply:
```suggestion
     * Make the current keyboard resizable.
     * See https://grammy.dev/plugins/keyboard.html#resize-custom-keyboard
     * for more details.
     *
     * Keyboards are resizeable by default, pass false to isEnable to make them non-resizable.
     *
     * @param isEnabled Determines if the keyboard should be resizable.
```

Reply:
This is exactly the opposite of what actually happens, but I like the idea of simply explaining the behaviour in sufficient depth here.

Reply:
My bad. @WingLim watch out for this, don't accept my suggestion.

Reply:
I make all of them to this, should be clear enought.

```
@param isEnabled Determine if the keyboard should be resizeable.
Default to `true`, you can pass `false` to disable it.
```

Reply:
We should explicitly note that the keyboard is not resized by default (as @KnorpelSenf said in his comment). This description is a bit misleading

Reply:
It isn't clear what "defaults to `true`" means. It could mean that the keyboard always defaults to having `resizable: true` so it's not necessary to call the method. It could also mean that if you call the method, the argument will default to `true` if it is not set.

I understand how the code actually works, but we should take into account that not everyone knows how keyboards work.

Reply:
How about this:

```
/**
     * Make the current keyboard selective.
     * See https://grammy.dev/plugins/keyboard.html#selectively-send-custom-keyboards
     * for more details.
     * 
     * Keyboards are non-resizeable by default, use this function to enable it(without any param).
     * And you can pass `false` to `isEnabled` to force make them non-resizable.
     *
     * @param isEnabled Determines if the keyboard should be selective.
     */
```

Reply:
I like that!